cmd/motion/main.go

```diff
@@ -1,7 +1,36 @@
 package main
 
-import "fmt"
+import (
+	"context"
+	"os"
+	"os/signal"
+
+	"github.com/filecoin-project/motion"
+	"github.com/ipfs/go-log/v2"
+)
+
+var logger = log.Logger("motion/cmd")
 
 func main() {
-	fmt.Println("Hello world from motion!")
+	// TODO: add flags, options and all that jazz
+	_ = log.SetLogLevel("*", "INFO")
```

Comment:
this should be based on
```suggestion
	// TODO: add flags, options and all that jazz
	if os.Getenv("GOLOG_LOG_LEVEL") == "" {
		_ = log.SetLogLevel("*", "INFO")
	}
```